app/services/trivia_service.rb

```diff
@@ -0,0 +1,19 @@
+class TriviaService
+  class << self
+    def get_url(url)
+      parse_data(conn.get(url))
+    end
+
+    def trivia
+      get_url("api.php?amount=500&category=29&type=multiple")
+    end
+
+    def conn
+      Faraday.new("https://opentdb.com/")
+    end
+
+    def parse_data(response)
+      parsed = JSON.parse(response.body, symbolize_names: true)
+    end
+  end
+end
```

Comment:
Faraday accepts query params as an argument, so I think there's a cleaner way to do this. You could add a second argument to your get_url method called `params` that includes everything after the ? in the url and then use that info in your get request. https://lostisland.github.io/faraday/usage/

Reply:
I think it's probably fine for now and we can come back and refactor later, but it should be a quick change if you want to take care of it now too

Reply:
I think we can leave this for now and refactor if we want to add ability to play other categories. If you feel like it is a MUST HAVE change let me know and I can submit another commit. Should be an easy fix 👍🏻